openstack_controller/tests/fixtures/GET/image/v2/images/limit=1/response.json

```diff
@@ -0,0 +1,35 @@
+{
+    "images": [
```

Comment:
What do you think about adding another image to the instance so we can confirm the check will collect data for more than 1 image when using pagination?

Reply:
Sounds good, I added another image.